src/drone/agent/executor.rs

```diff
@@ -179,12 +179,18 @@ impl Executor {
 
                     while let Some(v) = stream.next().await {
                         match v {
-                            Ok(v) => {
-                                if let Some(message) = DroneStatsMessage::from_stats_message(&v) {
+                            Ok(v) => match DroneStatsMessage::from_stats_message(&v) {
+                                Some(message) => {
                                     nc.publish(&DroneStatsMessage::subject(&backend_id), &message)
                                         .await?;
                                 }
-                            }
+                                None => {
+                                    let message =
+                                        "failed to get stats (container may have been swept)";
+                                    tracing::warn!(message);
```

Comment:
Might be useful to log the backend id here as well

Reply:
Also, maybe we should make this an `info` instead of a `warn`? We expect this to happen once for every backend after they've stopped, right?